sdks/python/setup.py

```diff
@@ -30,8 +30,7 @@
 
 def get_version():
   global_names = {}
-  execfile(os.path.normpath('./apache_beam/version.py'),
-           global_names)
+  exec(open(os.path.normpath('./apache_beam/version.py')).read(), global_names)
```

Comment:
Makes sense, thank you.

@wikier I tried `from apache_beam.utils import version` and that fails with `cannot import name coders`. Maybe we also need to change in `apache_beam/__init__.py` for the import to work.